### ISSUE TITLE
Do you have plane about every decoder step time output r frames？

### ISSUE THREAD

**Issue by sunnnnnnnny** [NVIDIA/tacotron2#208]
the training speed is very slow on a 1080ti if r equal 1

**Comment by BogiHsu** [comment]
@sunnnnnnnny I've made some modification to support reduction factor, you can refer to my repo [here](https://github.com/BogiHsu/Tacotron2-PyTorch).

**Comment by pravn** [comment]
@BogiHsu Do you have some samples?
The reduction factor would lead to "blurry" speech - as its effectively a
downsampling operation - and will have to be cleaned up/postprocessed in a
subsequent step, with a network whose architecture (e.g. dropout) might
need tuning.
Praveen

pravn.wordpress.com

**Comment by BogiHsu** [comment]
@pravn I do noticed the problem you mentioned. Since I am still tuning parameters, there is no sample for now. I'll release some samples and a pretrained model ASAP.

**Comment by WhiteFu** [comment]
> @pravn I do noticed the problem you mentioned. Since I am still tuning parameters, there is no sample for now. I'll release some samples and a pretrained model ASAP.

How's the result sample sound quality?

**Comment by BogiHsu** [comment]
> > @pravn I do noticed the problem you mentioned. Since I am still tuning parameters, there is no sample for now. I'll release some samples and a pretrained model ASAP.
> 
> How's the result sample sound quality?

The prosody is good, and the quality is ok when r is more than 3.

**Comment by rafaelvalle** [comment]
Closing due to inactivity.
